src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,70 @@
+.recommended {
+  margin-bottom: 120px;
+
+  @include onTablet {
+    margin-bottom: 128px;
+  }
+
+  &__title {
+    margin-bottom: 48px;
+
+    @include title;
+
+    @include onTablet {
+      margin-bottom: 64px;
+    }
+  }
+
+  &__content {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+
+    @include onDesktop {
+      display: flex;
```

Comment:
You already defined 'display: flex;' for '&__content' on line 19, so you don't need to repeat it in the onDesktop media query. 🧹